cpp/log/cluster_state_controller_test.cc

```diff
@@ -103,6 +107,7 @@ class ClusterStateControllerTest : public ::testing::Test {
 
   ThreadPool pool_;
   shared_ptr<libevent::Base> base_;
+  StrictMock<MockContinuousFetcher> fetcher_;
```

Comment:
I think this thing is actually a bit racy (Travis failed because of that), maybe I should use a NiceMock instead? That's not really the bit being tested here, maybe just ignoring everything is the right thing to do?